src/server/lib/svg-banner.js

```diff
@@ -124,7 +124,9 @@ export function generateSvgBanner(usersList, options) {
         const imageLink = `<a xlink:href="${website.replace(
           /&/g,
           '&amp;',
-        )}" class="opencollective-svg" target="_blank" rel="nofollow sponsored" id="${user.slug}">${image}</a>`;
+        )}" class="opencollective-svg" target="_blank" rel=${doFollow ? 'sponsored' : 'nofollow sponsored'} id="${
```

Comment:
Missing quotes.

```suggestion
        )}" class="opencollective-svg" target="_blank" rel="${doFollow ? 'sponsored' : 'nofollow sponsored'}" id="${
```

=>

![image](https://github.com/user-attachments/assets/9775d4e0-b626-4bfe-98b4-57e21e99e543)

Please make sure to test your changes, the instructions to start the project are in the README.